includes/Core/Conversion_Tracking/Conversion_Event_Providers/WooCommerce.php

```diff
@@ -34,7 +33,7 @@ class WooCommerce extends Conversion_Events_Provider {
 	 * @return bool True if WooCommerce is active, false otherwise.
 	 */
 	public function is_active() {
-		return Plugin_Status::PLUGIN_STATUS_ACTIVE === Plugin_Status::get_plugin_status( 'woocommerce/woocommerce.php', 'https://woocommerce.com/' );
+		return class_exists( 'WooCommerce' );
```

Comment:
This has been updated to use the documented method @eugene-manuilov found in https://github.com/google/site-kit-wp/pull/10251#discussion_r1964219334